### DISCUSSION TITLE
How to add a dynamic item

### DISCUSSION THREAD

**Discussion by Robinrrr** [VirtusLab/strapi-plugin-navigation#165] [Q&A]
First of all, thank you so much for this incredible plugin! I love it.

After playing around with it a bit, I noticed that I couldn't find a way to add dynamic urls based on a slug.

For example:

![image](https://user-images.githubusercontent.com/13948416/156000509-fa31c2ac-31d0-49cf-b7b5-e7f2a642f5d8.png)

I want to create an item that uses the slug I pass to identify the blog article. Is this possible? 
Maybe there is another way to solve the problem?

I am grateful for any help!

**Comment by cyp3rius** [comment]
Hello @Robinrrr , let us check that :)

@ltsNotMike

**Reply by ziiw** [reply]
Any news on that ? :) Thanks
